### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by SauloGC** [Miaosi001/JWLibrary-YearTextEditor#1]
Hi, I installed the application on a Windows 10 64 bits, but when I run it, a error message appears and also when I try to make some change in the year text and click save:
_An unhandled exception occurred in the application. Index outside the bounds of the array._

`   System.IndexOutOfRangeException: Index was outside the bounds of the array.`
`   at JWLibrary_YearTextEditor.Form1.Form1_Load(Object sender, EventArgs e)`
`   at System.EventHandler.Invoke(Object sender, EventArgs e)`
`   at System.Windows.Forms.Form.OnLoad(EventArgs e)`
`   at System.Windows.Forms.Form.OnCreateControl()`
`   at System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)`
`   at System.Windows.Forms.Control.CreateControl()`
`   at System.Windows.Forms.Control.WmShowWindow(Message& m)`
`   at System.Windows.Forms.Control.WndProc(Message& m)`
`   at System.Windows.Forms.ScrollableControl.WndProc(Message& m)`
`   at System.Windows.Forms.Form.WmShowWindow(Message& m)`
`   at System.Windows.Forms.Form.WndProc(Message& m)`
`   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)`
`   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)`
`   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)`

![Erro](https://user-images.githubusercontent.com/82067739/140400545-a57377e9-1ace-44ff-98e0-356243d8051a.png)

**Comment by darioragusa** [comment]
It looks like they modified the file with some update, I'll fix it as soon as I can

**Comment by darioragusa** [comment]
I think I fixed it, let me know if it works

**Comment by SauloGC** [comment]
Now it worked perfectly. I performed several tests and it is working properly. I had to uninstall version 1.0.0 and install 1.0.1 because just installing the new one over the old one didn't work. But after removing the old one and installing the new one, it worked correctly.

Before I was making some adjustments to the text manually in the `YearTextDisplay.bundle.js` file, mainly to adjust the line wrapping in my language. Now with your application it is extremely easier. Thank you very much for your effort.

![JWLibrary-YearTextEditor_v1 0 1 ok](https://user-images.githubusercontent.com/82067739/140505649-7e01b524-b158-44d5-9344-b7cd33ce5cf3.png)
